src/constants/notifications.ts

```diff
@@ -149,7 +153,7 @@ export type Notifications = Record<NotificationId, Notification<any>>;
 export type NotificationDisplayData = {
   icon?: React.ReactNode;
   title: string; // Title for Toast, Notification, and Push Notification
-  body?: string; // Description body for Toast, Notification, and Push Notification
+  body?: ReactNode; // Description body for Toast, Notification, and Push Notification
```

Comment:
body has to be a string :( due to the push notification lib

Reply:
If you really need to use a ReactNode, can use `renderCustomBody`

Reply:
rip but then i have to custom style the whole thing right?

Reply:
yeah :( if you want to avoid that since we aren't really showing anything special outside of <Output> you can probably create a string using @tyleroooo 's number formatting functions